app/peopledepot/settings.py

```diff
@@ -160,7 +171,9 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.0/howto/static-files/
 
-STATIC_URL = "static/"
+STATIC_URL = "/static/"
+STATIC_ROOT = BASE_DIR / "static"
+
```

Comment:
I just want to make sure. Was something not working before making this change? If not, then you should leave it alone. We don't have a static files solution for production yet and once we do, we'll need to change this to integrate that, so no need to do it now if nothing is complaining.